ElectronClient/gui/NoteText.jsx

```diff
@@ -1181,10 +1186,12 @@ class NoteTextComponent extends React.Component {
 		}
 
 		if (command.name === 'focusElement' && command.target === 'noteBody') {
-			fn = () => {
```

Comment:
Please put this back. If you don't know what some code does please ask, and don't remove stuff without really understanding what you're doing.

Reply:
Sorry for that. I'll just put that back real quick too.

Reply:
Done.